docs/quantized_ops.md

```diff
@@ -0,0 +1,114 @@
+Quantized Operations for XLA device (Experimental feature)
+--------------------------
+
+This document outlines how to utilize quantized operations to enable quantization on XLA devices.
+
+XLA Quantized ops offer a high-level abstraction for quantized operations (e.g., blockwise int4 quantized matrix multiplication). These ops are analogous to quantized CUDA kernels ([example](https://github.com/vllm-project/vllm/blob/main/csrc/quantization/gptq/q_gemm.cu)) in the CUDA ecosystem, providing similar functionality and performance benefits within the XLA framework.
+
+**NOTE:** Currently this is classified as experimental feature. It's API specifics 
+will change in the next (2.5) release.
+
+
+## How to use:
+
+XLA quantized operations can be used as `torch op`, or a `torch.nn.Module` that wraps the `torch.op`. These 2 options give model developers the flexibility to choose the best way to integrate XLA quantized ops into their solution.
+
+Both `torch op` and `nn.Module` are compatible with `torch.compile( backend='openxla')`.
+
+### Call XLA quantized op in model code
+
+Users can call XLA quantized ops in the same way as calling other regular PyTorch ops. This provides maximum flexibility in integrating XLA quantized ops into their applications. The quantized ops work in both eager mode and Dynamo, with regular PyTorch CPU tensor and XLA tensor.
+
+**Note** Please check the docstring of the quantized ops for the layout of the quantized weights.
+
+```Python
+import torch
+import torch_xla.core.xla_model as xm
+import torch_xla.experimental.xla_quantized_matmul
+
+N_INPUT_FEATURES=10
+N_OUTPUT_FEATURES=20
+x = torch.randn((3, N_INPUT_FEATURES), dtype=torch.bfloat16)
+w_int = torch.randint(-128, 127, (N_OUTPUT_FEATURES, N_INPUT_FEATURES), dtype=torch.int8)
+scaler = torch.randn((N_OUTPUT_FEATURES,), dtype=torch.bfloat16)
+
+# Call with torch CPU tensor
+matmul_output = torch.ops.xla.quantized_matmul(x, w_int, scaler)
```

Comment:
why do we want to show the cpu tensor part? Do we ever expect user to use it with CPU tensor?

Reply:
Yes, the is the ux for run the op in eager mode, it makes debugging easier I think

Reply:
please update the comment to say something like `this is mostly for debug`

Reply:
Done